backend/bitsurance/bitsurance.go

```diff
@@ -35,38 +37,41 @@ const (
 	widgetTestURL = "https://test.bitsurance.eu/?wallet=bitbox&version=" + widgetVersion + "&lang="
 )
 
-const statusActive = "active"
+type DetailStatus string
```

Comment:
exported things should have a docstring - I thought our CI would complain about it :smile: 

would be great to have a description here and for all the instances below about what the states mean and when they occur.

Reply:
It used to complain, don't know what happened :joy: 

I accidentally erased the comment explaining the possible status, I'll add it again.

Reply:
Done